src/js/utils/permissionUtils.js

```diff
@@ -41,4 +42,13 @@ const canEditRequest = (currentUser, request, location) => {
   return isLocationDestination && isUserRequestor;
 };
 
+export const hasRole = (user, role) => user?.roles?.includes(role);
+
+// To have permissions for product source list actions
+// user has to be at least admin and has product manager permissions
+export const hasPermissionsToProductSourceActions = (user, isAdmin) => {
+  const hasProductManagerPermission = hasRole(user, RoleType.ROLE_PRODUCT_MANAGER);
+  return hasProductManagerPermission && isAdmin;
+};
+
 export default canEditRequest;
```

Comment:
doesn't it break anything? It made sense when `canEditRequest` was the only method there.
I assume it doesn't, because you de-structured the object.